modules/react/button/lib/BaseButton.tsx

```diff
@@ -114,200 +111,262 @@ export const buttonVars = {
+      end: {},
     },
-    */
-  },
-});
-
-/**
- * Button modifiers that will overwrite the base styles of Buttons.
- * - `Size`: These modifiers will dictate a size of a Button and has a set of styles to associated with it.
- * - `iconPosition`: These modifiers will override the existing `Size` styles. These are specific to icon locations
- * within a button or if there is only an icon and no text.
- */
-export const buttonModifiers = createModifiers({
-  size: {
-    large: createStyles({
-      fontSize: system.space.x4,
-      lineHeight: system.space.x6,
-      letterSpacing: '0.01rem',
-      height: '48px',
-      paddingInline: system.space.x8,
-      minWidth: '112px',
-    }),
-    medium: createStyles({
-      fontSize: '0.875rem',
-      letterSpacing: '0.015rem',
-      minWidth: '96px',
-      paddingInline: system.space.x6,
-      height: system.space.x10,
-    }),
-    small: createStyles({
-      fontSize: '0.875rem',
-      letterSpacing: '0.015rem',
-      height: system.space.x8,
-      minWidth: system.space.x20,
-      paddingInline: system.space.x4,
-      gap: system.space.x1,
-    }),
-    extraSmall: createStyles({
-      fontSize: '0.75rem',
-      lineHeight: system.space.x4,
-      letterSpacing: '0.02rem',
-      height: system.space.x6,
-      minWidth: 'auto',
-      paddingInline: system.space.x3,
-      gap: system.space.x1,
-    }),
-  },
-  iconPosition: {
-    largeOnly: createStyles({
-      padding: '0',
-      minWidth: `calc(${system.space.x4} * 3)`,
-    }),
-    largeStart: createStyles({
-      paddingInlineStart: system.space.x6,
-      paddingInlineEnd: system.space.x8,
-    }),
-    largeEnd: createStyles({
-      paddingInlineStart: system.space.x8,
-      paddingInlineEnd: system.space.x6,
-    }),
-    mediumOnly: createStyles({padding: '0', minWidth: system.space.x10}),
-    mediumStart: createStyles({
-      paddingInlineStart: `calc(${system.space.x1} * 5)`,
-      paddingInlineEnd: system.space.x6,
-    }),
-    mediumEnd: createStyles({
-      paddingInlineStart: system.space.x6,
-      paddingInlineEnd: `calc(${system.space.x1} * 5)`,
-    }),
-    smallOnly: createStyles({padding: '0', minWidth: system.space.x8}),
-    smallStart: createStyles({
-      paddingInlineStart: system.space.x3,
-      paddingInlineEnd: system.space.x4,
-    }),
-    smallEnd: createStyles({
-      paddingInlineStart: system.space.x4,
-      paddingInlineEnd: system.space.x3,
-    }),
-    extraSmallOnly: createStyles({padding: '0', minWidth: system.space.x6}),
-    extraSmallStart: createStyles({
-      paddingInlineStart: system.space.x2,
-      paddingInlineEnd: system.space.x3,
-    }),
-    extraSmallEnd: createStyles({
-      paddingInlineStart: system.space.x3,
-      paddingInlineEnd: system.space.x2,
-    }),
   },
+  // Compound Modifier Styles
+  compound: [
+    {
+      modifiers: {size: 'large', iconPosition: 'only'},
+      styles: {
+        minWidth: `calc(${system.space.x4} * 3)`,
```

Comment:
`minWidth: `calc(${system.space.x4} * 3)`,`

Can we use `calc.multiply`? And what is the better approach?

Reply:
We can use `calc.multiply()`. 

Just went ahead and updated it.